pom.xml

```diff
@@ -2023,6 +2023,46 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                    <groupId>com.github.ekryd.sortpom</groupId>
+                    <artifactId>sortpom-maven-plugin</artifactId>
+                    <version>3.2.1</version>
+                    <executions>
+                        <execution>
+                            <id>sortpom-warn</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>verify</goal>
+                            </goals>
+                            <configuration>
+                                <verifyFail>Warn</verifyFail>
+                            </configuration>
+                        </execution>
+                        <execution>
+                            <id>sortpom-fail</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>verify</goal>
+                            </goals>
+                            <configuration>
+                                <verifyFail>Stop</verifyFail>
+                            </configuration>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <createBackupFile>false</createBackupFile>
+                        <keepBlankLines>true</keepBlankLines>
+                        <nrOfIndentSpace>4</nrOfIndentSpace>
+                        <sortOrderFile>${air.main.basedir}/.mvn/sortpom/sortorder.xml</sortOrderFile>
```

Comment:
Let's delete this (and `sortorder.xml`) and use the default `<predefinedSortOrder>recommended_2008_06</predefinedSortOrder>` which is the Maven standard and seems to match my expectations (it looks like how I would order a POM file myself).

Reply:
Returned to default sortorder.